src/lib.rs

```diff
@@ -24,15 +24,36 @@ pub fn package(name: &str) -> Option<&cargo_metadata::Package> {
     metadata().packages.iter().find(|x| x.name == name)
 }
 
+fn build_dir_path(path: Option<PathBuf>, release: bool) -> PathBuf {
+    path.unwrap_or_else(|| {
```

Comment:
1. People can do the unwrap_or_else on their own.
2. Make this public
3. Use camino as cargo_metadata uses camino anyway (re-exporting camino wouldn't be a bad idea too)
4. Use lazy_static to avoid re-computation

```suggestion
pub fn default_build_dir(release: bool) -> &Utf8Path {
```